python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -503,20 +519,27 @@ class RandomForestClassifier(Base):
         del(y_m)
         return self
 
-    def predict(self, X):
-        """
-        Predicts the labels for X.
-        Parameters
-        ----------
-        X : array-like (device or host) shape = (n_samples, n_features)
-            Dense matrix (floats or doubles) of shape (n_samples, n_features).
-            Acceptable formats: cuDF DataFrame, NumPy ndarray, Numba device
-            ndarray, cuda array interface compliant array like CuPy
-        Returns
-        ----------
-        y: NumPy
-           Dense vector (int) of shape (n_samples, 1)
-        """
+    def predict_model_on_gpu(self, X, output_class,
```

Comment:
This should probably be a “private” method starting with `_` since users will call predict and predict will call this
```suggestion
    def _predict_model_on_gpu(self, X, output_class,
```